force_wfmanager/ui/setup/process/data_source_view.py

```diff
@@ -39,12 +41,15 @@ class TableRow(HasStrictTraits):
     description = Unicode()
 
     def __init__(self, model, *args, **kwargs):
+        #: FIXME: Child model should not be instantiated before super
```

Comment:
Moving the `self.model = model` to after the super instantiation breaks the constructor. I don't know why, but this should be looked at in the future